ReportUtilityBills/measure.rb

```diff
@@ -169,7 +169,7 @@ def energyPlusOutputRequests(runner, user_arguments)
     @hpxml_buildings = hpxml.buildings
     if @hpxml_header.utility_bill_scenarios.has_detailed_electric_rates
       uses_unit_multipliers = @hpxml_buildings.select { |hpxml_bldg| hpxml_bldg.building_construction.number_of_units > 1 }.size > 0
-      if uses_unit_multipliers || (@hpxml_buildings.size > 1 && building_id == 'ALL')
+      if uses_unit_multipliers || (@hpxml_buildings.size > 1 && !hpxml.header.whole_sfa_or_mf_building_sim)
```

Comment:
Should this be `hpxml.header.whole_sfa_or_mf_building_sim` (without the "!")?

Reply:
Yep, it should. I noticed that I got it wrong when re-reviewing, which is what got me down this rabbit hole. And there were no tests for this (since all tests passed), so I'm adding one.